dev/tools/merge-pr.sh

```diff
@@ -1,50 +1,156 @@
 #!/usr/bin/env bash
 
 set -e
+set -o pipefail
+
+API=https://api.github.com/repos/coq/coq
+OFFICIAL_REMOTE_URL="git@github.com:coq/coq.git"
 
 # This script depends (at least) on git and jq.
 # It should be used like this: dev/tools/merge-pr.sh /PR number/
```

Comment:
This info predates this PR but it is incomplete.

Reply:
ok